Makefile

```diff
@@ -35,7 +35,7 @@ all: check-style test dist
 ## Propagates plugin manifest information into the server/ and webapp/ folders.
 .PHONY: apply
```

Comment:
Do we need to keep this around at all? This change only applies to new repositories, or opt-in to the updated starter-template, and we don't guarantee any kind of backwards compatibility today. I'd love to push this repository towards being "legacy free".

If we /do/ want it, might we introduce a `build/legacy.mk` to host these kinds of things?

Reply:
The plugin ci orb has a check for the generated files (https://github.com/mattermost/circleci-orbs/blob/3236f246b4d9d9a766646d1a2442fff45ab55fcf/plugin-ci/orb.yml#L84-L88), that would fail if `make apply` is not found. We could just append `||  true` and fix it that way, but I would like to keep the check for plugins that didn't dropped the generated files yet. This was the main reason I've kept `make apply`.

:+1: for using a `build/legacy.mk` is we want to keep the command.